server/upgrade.js

```diff
@@ -0,0 +1,70 @@
+const mongo   = require("promised-mongo")
+const db      = mongo("mongodb://localhost/bydesign", ["authors", "posts", "metadata", "categories"])
+const console = require("beautiful-log")
```

Comment:
I suppose next version you could do

```js
let log = require("beautiful-log");
log.init("appname");
console = log.make("logger-name");
```

I don't like patching over builtins if it can be avoided.  That way you can keep the other console methods.